contributors/design-proposals/local-storage-overview.md

```diff
@@ -101,22 +103,22 @@ Since local PVs are only accessible from specific nodes, the scheduler needs to
      volumes:
      - name: myEmptyDir
        emptyDir:
-	     sizeLimit: 20Gi
+         sizeLimit: 20Gi
     ```
 
 3. Alice’s pod “foo” is Guaranteed a total of “21.5Gi” of local storage. The container “fooc” in her pod cannot consume more than 1Gi for writable layer and 500Mi for logs, and “myEmptyDir” volume cannot consume more than 20Gi.
 4. For the pod resources, `storage.kubernetes.io/logs` resource is meant for logs. `storage.kubernetes.io/writable` is meant for writable layer.
 5. `storage.kubernetes.io/logs` is satisfied by `storage.kubernetes.io/scratch`.
 6. `storage.kubernetes.io/writable` resource can be satisfied by `storage.kubernetes.io/overlay` if exposed by nodes or by `storage.kubernetes.io/scratch` otherwise. The scheduler follows this policy to find an appropriate node which can satisfy the storage resource requirements of the pod.
```

Comment:
@jingxu97 Are we still using `storage.kubernetes.io/writable` as a resource type?  What's the relationship between `storage.kubernetes.io/overlay`?